Download Initial Data

gwaybio opened the pull request:
I shared this script on slack previously, but I think it would work well as the tipping off point for this repository. 

Note that the script doesn't actually work right now :roll_eyes: - there is something buggy about GEO ftp. Hopefully this gets squared away soon. We can also wait to merge this until it works if people feel strongly.

I also added `data/` to the `.gitignore`. My thoughts on what to do with the data are in #1  - we can discuss there alternative options too.

gwaybio committed:
convert license to markdown

gwaybio committed:
remove license file

gwaybio committed:
add initial download script

gwaybio committed:
ignore data directory

gwaybio commented:
We should also discuss how we should handle PR review too @huqiwen0313 @jaclyn-taroni

gwaybio committed:
change to BSD-3-Clause name

gwaybio committed:
update to BSD-3-Clause text

gwaybio committed:
add CC-BY license

jaclyn-taroni commented on download_data.sh:
```diff
@@ -0,0 +1,45 @@
+# expression analyses. The script will fetch, decompress, and rename each data set
+# listed below
+
+mkdir --parents data/raw
```
Should this be followed by `cd data && cd raw`?

gwaybio replied:
i process files in this script too - it moves the data to appropriate directory

jaclyn-taroni replied:
Oh yea, my bad

jaclyn-taroni reviewed (approved):
One comment/question about directories in the data download shell script, otherwise LGTM 👍

huqiwen0313 commented on download_data.sh:
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
```
Is it possible to add some descriptions for these samples or have a file described what these samples are.

gwaybio replied:
yeah, I think that is a good idea. I will add a brief description here, but a more thorough description should exist as some sort of table, text, or FAQ in the README or github wiki

gwaybio commented:
> Note that the script doesn't actually work right now :roll_eyes: - there is something buggy about GEO ftp. 

Can confirm that it now works!

gwaybio committed:
add brief description of datasets

also remove meyer et al

gwaybio commented:
Thanks @huqiwen0313 - I added a brief description to the data in the current download file. Also note that this script should have a couple datasets also added to it (in #3 and #4)
